src/status_im2/common/scroll_page/view.cljs

```diff
@@ -54,17 +54,6 @@
                :top      0
                :left     0
                :right    0}}
-      (when logo
```

Comment:
it's okay to delete this? this component is used in the communities overview page? or am I missing something and this code is elsewhere now?

Reply:
`scroll-page` uses `page-nav` internally but before it was using it with `:type :no-title` and showing title/logo on top of that component. Since `page-nav` know how to render title and logo, I got rid of logo in title directly in `scroll-page` and added the possibility to configure internal `page-nav` instead. Here is how communities overview now configured:
https://github.com/status-im/status-mobile/blob/6eeff34d9371e0b9d67e9b66f95422a189baccaa/src/status_im2/contexts/communities/overview/view.cljs#L329

Reply:
Thanks for the explanation! 🙏